packages/tpetra/core/inout/MatrixMarket_Tpetra.hpp

```diff
@@ -3840,18 +3840,36 @@ namespace Tpetra {
           values = null;
         } // if myRank == rootRank
 
+        //FIXME this is no longer needed
         broadcast<int,size_t> (*pComm, 0, &maxNumEntriesPerRow);
```

Comment:
The comment says, this is not need anymore. Can it be removed?

Reply:
Fixed with commit 4a60479.